.buildkite/pipelines/pull_request/security_solution.yml

```diff
@@ -5,10 +5,10 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 60
-    parallelism: 7
+    parallelism: 10
     retry:
       automatic:
-        - exit_status: '-1'
```

Comment:
What do `exit_status`s `-1` and `*` indicate? What is the effect of these changes?

Reply:
https://buildkite.com/docs/pipelines/command-step#retry-attributes-automatic-retry-attributes

Reply:
Answer: 
> -1 means buildkite agent connection was aborted. [docs](https://buildkite.com/docs/pipelines/command-step#retry-attributes-automatic-retry-attributes)

The effect here will be that we only retry these jobs once (instead of thrice) in the case of timeout (or of failure).